package.json

```diff
@@ -187,6 +187,28 @@
           "type": "boolean",
           "default": true,
           "markdownDescription": "When enabled, if you press the return key from the first line of a multiline string containing a pipe, it will automatically add `.stripMargin`."
+        },
+        "metals.fallbackScalaVersion": {
+          "type": "string",
+          "default": "",
```

Comment:
We should not have any default, otherwise I think empty string will be sent to the server.

Reply:
Thx, I forgot to remove it.

By the way, I remembered that there is a minor issue with this selector.
What do you think about adding an option that will be called `default`?

There is a logic that tries to select the highest Scala version from build targets. 
It applies in case if this property is empty. But `enum` type doesn't allow you to choose empty so there will be no way to clean up this property.

Reply:
Maybe something like `automatic` ?

Reply:
Added `automatic`.

Also mentioned it in the description section. @ckipp01 could you check grammar there one more time?)

Reply:
Sorry, late to the party on this. I hate to say this now, but what happens if you just don't have a default, what does VS Code send? In some ways, even if it is an empty string, I'd almost rather that, and instead have Metals just filter out an empty string. Mainly because now in this meaningful list of Scala versions we have a random `automatic`. It's not the worst, but just treating `""` as None on the Metals side instead of introducing another value makes more sense to me. It's not a hill I'll die but just my two cents.

Reply:
@ckipp01
In VS Code `enum` option can't be manually edited. So `automatic` value solves the problem when you might decide to set `None` value back. Otherwise, you will have to manually edit `.vscode/settings.json`.

It might represented be just `""` but I'm afraid it will more confusing than `automatic` or smth like that.
Unfortunately, there is no way to specify how options should be rendered in VS Code selector. Only plain values.

> Does this mean that if I have set 2.12 set for one module and 2.13 set for another, you'll choose 2.13? I'm just afraid that this may cause confusion

Yes, it will select 2.13. 
I don't think that is confusing because most of the users use a single Scala version per project and will never face the need to change this option.

Reply:
For an Ammonite script, it will be Scala 2 because it doesn't support scala 3 yet.
For a worksheet or scala file, it will be Scala 3